assets/js/components/user-input/UserInputPreviewGroup.js

```diff
@@ -211,7 +211,10 @@ export default function UserInputPreviewGroup( {
 
 							<div className="googlesitekit-user-input__preview-actions">
 								<SpinnerButton
-									disabled={ ! hasSettingChanged }
+									disabled={
+										! hasSettingChanged ||
+										values.length === 0
+									}
```

Comment:
Why was this condition added?

Reply:
Thanks for spotting it. I mistakenly included/mixed it from other task. Removed.